static/app/views/dashboardsV2/detail.tsx

```diff
@@ -429,22 +392,61 @@ class DashboardDetail extends Component<Props, State> {
   }
 
   render() {
-    const {api, location, params, organization} = this.props;
+    const {organization, dashboard, dashboards, params, router, location} = this.props;
+    const {modifiedDashboard, dashboardState} = this.state;
+    const {dashboardId} = params;
+
+    if (this.isEditing && this.isWidgetBuilderRouter) {
+      return this.renderWidgetBuilder(dashboard);
+    }
 
     return (
-      <OrgDashboards
-        api={api}
-        location={location}
-        params={params}
-        organization={organization}
-      >
-        {({dashboard, dashboards, error, reloadData}) => {
-          if (this.isEditing && this.isWidgetBuilderRouter) {
-            return this.renderWidgetBuilder(dashboard);
-          }
-          return this.renderDetails({dashboard, dashboards, error, reloadData});
-        }}
-      </OrgDashboards>
+      <Feature features={['dashboards-basic']} organization={organization}>
```

Comment:
should we also check the` dashboards-basic `feature flag when rendering the widget builder?

Reply:
Hmm I think we might want to do `dashboards-edit` as the feature flag here right? because without the edit feature they won't be able to add/edit widgets?